charts/gaps/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for Gaps.
+
+image:
+  repository: housewrecker/gaps
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
Are there no versioned images for this? 🥺

Reply:
On my list to create a image. Maintainer doesn't follow semver well and nit multiarch images

Reply:
But we can use this as is for now until i get to that

Reply:
Cool!